sql/introduction-to-statistics.md

```diff
@@ -0,0 +1,98 @@
+---
+title: 统计信息简介
+category: user guide
+---
+
+# 统计信息简介
+
+TiDB 优化器会根据统计信息来选择最优的执行计划。统计信息收集了表级别和列级别的信息，表的统计信息包括总行数，以及修改的行数。列的统计信息包括不同值的数量，NULL 的数量，以及该列的直方图信息。
+
+## 统计信息的收集
+
+### 手动收集
+
+你可以通过执行 `ANALYZE` 语句来收集统计信息。
+
+语法：
+```sql
+ANALYZE TABLE TableNameList
+> 该语句会收集 TableNameList 中所有表的统计信息。
+
+ANALYZE TABLE TableName INDEX IndexNameList
+> 该语句会收集 TableName 中所有的 IndexNameList 中的索引列的统计信息。
+```
+
+### 自动更新
+
+在发生增加，删除以及修改语句时，TiDB 会自动更新表的总行数以及修改的行数。这些信息会定期持久化下来，更新的周期可以由 `statsLease` 参数指定，
```

Comment:
....why an upper case L here?
Should TiDB fix this first?

Reply:
In the config file, this has changed to `stats-lease`.